### ISSUE TITLE
Codecov step 3.3.0 : error xcrun is not installed

### ISSUE THREAD

**Issue by anthony-fresneau-kiplin** [codecov/codecov-bitrise#33]
Hi, we meet the following error  with step 3.3.0 on an Android project: 
[2023-08-03T07:48:04.046Z] ['error'] There was an error running the uploader: xcrun is not installed, cannot process files
How to fix it ?

**Comment by thomasrockhu-codecov** [comment]
@anthony-fresneau-kiplin fix is pending on `3.3.1` [here](https://github.com/bitrise-io/bitrise-steplib/pull/3932)

**Comment by anthony-fresneau-kiplin** [comment]
maybe the pr for the fix could be submitted here(on this repo) ?

**Comment by thomasrockhu-codecov** [comment]
@anthony-fresneau-kiplin that won't actually release the Bitrise step, it needs to be merged into this repo via [this PR](https://github.com/bitrise-io/bitrise-steplib/pull/3932)

**Comment by thomasrockhu-codecov** [comment]
@anthony-fresneau-kiplin we released `3.3.1` earlier today. Can you see if this works for you?
